tests/test_pandas_extension.py

```diff
@@ -128,7 +128,22 @@ def choices(seq, k):
 ]
 
 
-@pytest.fixture(params=test_types)
+def get_parameter_ids(test_types):
```

Comment:
This seems unused. For what do you use this?

Reply:
This sets the parameter IDs for the tests. The test parametrization then shows e.g. `Date(string)` instead of `fletcher_type_0`
For usage see a line 143